packages/styles/framework/src/components/_flex-box.scss

```
@@ -35,7 +35,10 @@
   }
 
   @include mobile {
-    &:has(>.flex-item):not([class*='is-direction']) {
+    &:has(>.flex-item):not([class*='is-direction']):not(.is-mobile) {
```

Stylelint (17.2.0)

[error] Expected complex :not() pseudo-class notation (selector-not-notation)

(selector-not-notation)